packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/apis/update.ts

```diff
@@ -246,18 +246,24 @@ export const executeUpdate = async <T>(
     // at this point, we already know 1. the document exists 2. we're not doing an upsert
     // therefor we can safely process with the "standard" update sequence.
 
-    const updatedAttributes = mergeForUpdate({
-      targetAttributes: {
-        ...(migrated!.attributes as Record<string, unknown>),
-      },
-      updatedAttributes: await encryptionHelper.optionallyEncryptAttributes(
-        type,
-        id,
-        namespace,
-        attributes
-      ),
-      typeMappings: typeDefinition.mappings,
-    });
+    const mergeAttributes = options.mergeAttributes ?? true;
```

Comment:
NIT consistency: All other options are extracted on lines 87-93, and the optional ones have default values that are stored in `../constants.ts`.